build.gradle

```diff
@@ -315,8 +315,7 @@ task fullAcceptanceTest(type: GradleBuild) {
 task acceptanceTest(type: Test) {
     testClassesDir = sourceSets.acceptanceTest.output.classesDir
     classpath = sourceSets.acceptanceTest.runtimeClasspath
-//    maxParallelForks = Runtime.runtime.availableProcessors()
-    maxParallelForks = 1
+    maxParallelForks = Runtime.runtime.availableProcessors()
```

Comment:
I decided to get rid of commented line (cause I do not like commented lines) and for me it works pretty stable and much faster then single-threaded version. 
For travis it didn't change anything.